<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VolumeRenderer</class>
 <widget class="QMainWindow" name="VolumeRenderer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Volume Renderer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpenImage"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_About"/>
   </widget>
   <widget class="QMenu" name="menuRenderer">
    <property name="title">
     <string>Renderer</string>
    </property>
    <addaction name="action_vtkGPUVolumeRayCastMapper"/>
    <addaction name="action_vtkSlicerGPURayCastVolumeMapper"/>
   </widget>
   <widget class="QMenu" name="menuScreenshot">
    <property name="title">
     <string>Screenshot</string>
    </property>
    <addaction name="actionScreenshotViewPane"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menuRenderer"/>
   <addaction name="menuScreenshot"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget">
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2"/>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="action_Open">
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="action_vtkGPUVolumeRayCastMapper">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>vtkGPUVolumeRayCastMapper</string>
   </property>
   <property name="toolTip">
    <string>vtkGPUVolumeRayCastMapper</string>
   </property>
  </action>
  <action name="action_vtkSlicerGPURayCastVolumeMapper">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>vtkSlicerGPURayCastVolumeMapper</string>
   </property>
  </action>
  <action name="actionScreenshotViewPane">
   <property name="text">
    <string>ScreenshotViewPane</string>
   </property>
  </action>
  <action name="actionOpenImage">
   <property name="text">
    <string>OpenImage</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
